packages/katello/rubygem-pulp_container_client/rubygem-pulp_container_client.spec

```
@@ -1,26 +1,23 @@
-# Generated from pulp_docker_client-4.0.0b6.dev01562331743.gem by gem2rpm -*- rpm-spec -*-
 # template: scl
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
-%global gem_name pulp_docker_client
+%global gem_name pulp_container_client
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 4.0.0b6.dev01562331743
+Version: 1.0.0rc1
 Release: 1%{?dist}
-Summary: Pulp 3 Docker API Ruby Gem
+Summary: Pulp container plugin for Pulp3 API Ruby Gem
 Group: Development/Languages
-License: GPLv2
-URL: https://rubygems.org/gems/pulp_docker_client
+License: GPL-2.0
```

Looks like the license is missing in https://github.com/theforeman/foreman-packaging/blob/rpm/develop/licenses.json

```suggestion
License: GPLv2
```